lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -428,7 +431,7 @@ defmodule ExUnit.DocTest do
   defp extract_tests(line_no, doc, module) do
     all_lines = String.split(doc, ~r/\n/, trim: false)
     lines = adjust_indent(all_lines, line_no + 1, module)
-    extract_tests(lines, "", "", [], true)
+    do_extract_tests(lines, "", "", [], true, module)
```

Comment:
We only do that if we need to implement a private function with exact the same arity. If the arity is not taken, then we go without the `do_`. And @lexmag and I have been discussing to find ways to avoid the `do_` altogether and with with more describe names, such as `extract_tests_with_acc`.